matrix-sdk/src/main/java/org/matrix/androidsdk/HomeServerConnectionConfig.java

```diff
@@ -138,6 +143,23 @@ public Uri getIdentityServerUri() {
         return (null == mIdentityServerUri) ? mHsUri : mIdentityServerUri;
     }
 
+    /**
+     * Update the anti-virus server URI.
+     *
+     * @param uri the new anti-virus uri
+     */
+    public void setAntiVirusServerUri(Uri uri) {
+        mAntiVirusServerUri = uri;
+    }
+
+    /**
+     * @return the anti-virus server uri
+     */
+    public Uri getAntiVirusServerUri() {
+        // Consider the HS uri by default.
+        return (null == mAntiVirusServerUri) ? mHsUri : mAntiVirusServerUri;
```

Comment:
I do not like the ternary operator `?` cause it may lead to code not easy to read.

You can keep it here, but I think this code is easier to read (inverse if condition to put fallback has the last statement):
`return mAntiVirusServerUri != null ? mAntiVirusServerUri : mHsUri;`